README.md

```diff
@@ -43,8 +43,7 @@ eval $(minikube docker-env)
 
 # On Windows minikube mount doesn't work out of the box ("minikube --mount-string" is problematic).
 # Open another terminal in the same folder and run
-minikube ip  # Get IP of the minikube machine (192.168.99.100 by default)
-minikube mount .:/src/sample-app --ip=192.168.99.100  # Fill in the correct IP if different.
+minikube mount .:/src/sample-app --ip=192.168.99.1
```

Comment:
Why do you need to change ip here? It is better to mention `minikube ip`, than hard-coding your env specific ip. Like it was before.

Reply:
Needed to change the IP to that exact static value on Windows as the minikube ip address does not work there. It can only find and use that IP stated there.